astro.config.mjs

```diff
@@ -19,12 +21,53 @@ import tqlLang from './tql.tmLanguage.json' assert { type: 'json' };
 const runLinkCheck = process.env.RUN_LINK_CHECK || false;
 const isProd = process.env.NODE_ENV === 'production';
 
+// Function to recursively find all .md files in a directory
+function findMarkdownFiles(dir, baseDir) {
+  const files = [];
+  const items = readdirSync(dir);
+  
+  for (const item of items) {
+    const fullPath = join(dir, item);
+    const stat = statSync(fullPath);
+    
+    if (stat.isDirectory()) {
+      files.push(...findMarkdownFiles(fullPath, baseDir));
+    } else if (item.endsWith('.md')) {
+      const relativePath = relative(baseDir, fullPath);
+      files.push(relativePath.replace(/\.md$/, ''));
```

Comment:
Have you checked whether we also have *.mdx files? AFAICT not in the reference, but I didn't check.

Reply:
Done in ad7d2beebee5a6b34bd64ed770f8192807bc30cb